components/global/history_buttons/history_buttons.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+import IconButton from '@mattermost/compass-components/components/icon-button';
+
+import {trackEvent} from 'actions/telemetry_actions';
+import * as Utils from 'utils/utils';
+import {isDesktopApp} from 'utils/user_agent';
+import {browserHistory} from 'utils/browser_history';
+
+const HistoryButtonsContainer = styled.nav`
+    display: flex;
+    align-items: center;
+`;
+
+const HistoryButtons = (): JSX.Element | null => {
+    if (!isDesktopApp()) {
+        // need to re-enable this
```

Comment:
?? Guessing this should be disabling in the desktop app.

Reply:
ahh this is just so that UX or QA can see it in the test server, it will be removed after the test is done. These history arrows should only show in desktop app.